webpack/assets/javascripts/react_app/routes/FiltersForm/FiltersForm.js

```diff
@@ -117,7 +130,29 @@ export const FiltersForm = ({ roleName, roleId, isNew, data, history }) => {
         defaultPermissions={data.permissions}
         setChosenPermissions={setChosenPermissions}
       />
-      {(showOrgs || showLocations) && (
+      <FormGroup label={__('Organizations')}>
+        <TextInput
+          ouiaId="filter-org-text-disabled"
+          aria-label="disabled organization field"
+          isDisabled
+          type="text"
+          value={chosenOrgs.join(' ,')}
+        />
+      </FormGroup>
+
+      <FormGroup
+        label={__('Locations')}
+        helperText={__('Taxonomies are inherited from the role')}
```

Comment:
Instead of "Taxonomies are inherited...", how about adding a line like that after Organizations and Locations each? So: "Organizations are inherited..." and "Locations are inherited ...". It's two lines instead of one but I'm concerned about the term taxonomies: will everyone know that it refers to orgs and locs?

Also, looking at the web UI, due to the layout of the page (indentation) and the formatting, it might look like "Taxonomies are inherited..." relates only to Locations.

Reply:
Yeah, thats a better way, I'll change it